File: testdata/changeSignature/fromJava/ArrayToVarargs.scala
class ArrayToVarargsTest extends ArrayToVarargs {
  override def foo(i: Int, js: Array[Int], b: Boolean): Unit = super.foo(i, js, b)

  foo(1, Array(2, 2), true)
  val xs = Array(3, 3)
  foo(1, xs, true)
}

File: scala/scala-impl/testdata/changeSignature/fromJava/ArrayToVarargs.java
public class ArrayToVarargs {
    public void <caret>foo(int i, int[] js, boolean b) {}

    public void test() {
        foo(1, new int[]{2, 2}, true);
        int[] xs = new int[]{3, 3};
        foo(1, xs, true);
    }
}